React/Modules/RCTUIManager.m

```diff
@@ -37,8 +37,10 @@
 #endif // TODO(macOS ISS#2323203)
 #import "RCTShadowView+Internal.h"
 #import "RCTShadowView.h"
+#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
```

Comment:
Are these necessary?   I can remove them locally and it work,s

Reply:
Yup. If you remove the iOS target from the `Podfile` the build will fail. In your test the header exists, even if its corresponding implementation file is only a member of the iOS build.

Perhaps more important, all references in this file to APIs in these headers have also been guarded.